MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetLoader.cs

```diff
@@ -292,7 +292,7 @@ internal void OnAssetUpdate(AssetUpdate payload, Action onCompleteCallback)
 				if (!_owner) return;
 
 				var mat = asset as UnityEngine.Material;
-				var tex = asset as UnityEngine.Texture;
+				var tex = asset as UnityEngine.Texture2D;
```

Comment:
If we keep this type as a `UnityEngine.Texture` we keep things open to add `RenderTexture` support later. Is there a reason this has to be a `Texture2D`?